week-3/implement/get-angle-type.js

```diff
@@ -21,3 +21,24 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+function getAngleType(angle) {
+  if (angle === 90) return "Right angle";
+  if (angle < 90) return "Acute angle";
+  if (angle === 180) return "Straight angle";
+  if (angle < 180) return "Obtuse angle";
+  if (angle === 360) return "Full angle";
+  if (angle < 360) return "Reflex angle";
+  return "Unknown angle";
+}
```

Comment:
I really like how you have wriiten your if statements in one line. It is easy to follow and easy to read. 
I like that you also have a statement for unknown angle. That is really good. :+1: :100:

Reply:
Thank you.